change/msal-node-client-credentials-2020-08-14-17-01-57-msal-node-client-credential-grant.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
This is a sample, should be "none" change type. We don't want to release samples.

Reply:
Updated